### DISCUSSION TITLE
Suggestion on WeedWallet

### DISCUSSION THREAD

**Discussion by kjincgroup** [BitCannaGlobal/WeedWallet#59] [Ideas]
1. Remove the Pro and Simple Toggle from other page, and only show in Portfolio page, since it only work on Portfolio page. 
(You can ignore this, if there are planned feature will be launched for each page)

2. Add a delegate option in Wallet delegation in Validators page, it will be more convenient  
![image](https://github.com/BitCannaGlobal/WeedWallet/assets/111372349/be3f541f-32f0-4dcc-a06e-354638751269)

3. Allow to change our own profile picture

4. Allow to switch background color, maybe a toggle to switch between black and white?

5. Allow to switch RPC or API Endpoint
![image](https://github.com/BitCannaGlobal/WeedWallet/assets/111372349/cf029f46-1e55-48cf-a52c-5b8ce4cf331e)

**Comment by TJ-BitCanna** [comment]
1) We are indeed already working on this. Our designer is working on a design which is in line with the mobile app we are working on, the "Simple" page is a first glimpse of it.

2) Good point, we'll assess and take it with us.

3) This we can't do. We don't have a database running behind the wallet, but it is completely decentralized. The profile picture is either a generic one for people not owning a Buddhead OR showing their Buddheads when they have them. So you can "choose" your profile picture by choosing a fun Buddhead you like.

4) Good point, something we can check for a later release.

5) What would be the added value of doing so?

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- Add a button on the Validator page to "delegate" coins directly without having to open the validator specific page.

**Comment by TJ-BitCanna** [comment]
In the new design which will be implemented before release this button will be included.

Issue will be closed.

**Comment by kjincgroup** [comment]
For point 5, Allow to switch RPC or API Endpoint. in case the endpoint is not stable or "dead",  it is good to allow user to switch different endpoint, like ping.pub
